### PR TITLE
fix debuginfo for windows_gnullvm_base.rs

### PR DESCRIPTION
**Pull request by jeremyd2019**
These lines (including the FIXME comment) were added to windows_gnu_base.rs in cf2c492ef8c87c049b4e3a62f43c841aafc88cba but windows_gnullvm_base.rs was not updated.  This resulted in an error `LLVM ERROR: dwo only supported with ELF and Wasm` attempting to build on aarch64-pc-windows-gnullvm.

See also https://github.com/msys2/MINGW-packages/pull/13921#issuecomment-1304391707

/cc @mati865 @davidtwco

r? @davidtwco

### CODE REVIEW

File: compiler/rustc_target/src/spec/windows_gnullvm_base.rs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-use crate::spec::{cvs, Cc, LinkerFlavor, Lld, TargetOptions};
+use crate::spec::{cvs, DebuginfoKind, LinkerFlavor, SplitDebuginfo, TargetOptions};
```
Comment (jeremyd2019) [inline compiler/rustc_target/src/spec/windows_gnullvm_base.rs:1]:
```suggestion
use crate::spec::{cvs, CC, DebuginfoKind, LinkerFlavor, Lld, SplitDebuginfo, TargetOptions};
```

Oops, manual merge issue

**Comment by jeremyd2019** [comment]
This is my first attempt to open a pull request against rust.  I don't even really use/understand rust, I just happen to be the person with the machine to build for this target at msys2.  Please have patience with me 😉

File: compiler/rustc_target/src/spec/windows_gnullvm_base.rs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-use crate::spec::{cvs, Cc, LinkerFlavor, Lld, TargetOptions};
+use crate::spec::{cvs, CC, DebuginfoKind, LinkerFlavor, Lld, SplitDebuginfo, TargetOptions};
```
Comment (jeremyd2019) [inline compiler/rustc_target/src/spec/windows_gnullvm_base.rs:1]:
```suggestion
use crate::spec::{cvs, Cc, DebuginfoKind, LinkerFlavor, Lld, SplitDebuginfo, TargetOptions};
```

**Comment by mati865** [comment]
The policy is to merge commits as they are so to keep clean history users are expected rebase + squash/fixup "fixup" commits.

**Review by davidtwco — APPROVED** [review]

**Pull request merged**